code_of_conduct.md

```diff
@@ -52,14 +52,15 @@ address, posting via an official social media account, or acting as an appointed
 representative at an online or offline event. Representation of a project may be
 further defined and clarified by project maintainers.
 
-## Enforcement
+## Enforcement / Reporting a Problem or Concern
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at join.ifme@gmail.com. All
-complaints will be reviewed and investigated and will result in a response that
-is deemed necessary and appropriate to the circumstances. The project team is
-obligated to maintain confidentiality with regard to the reporter of an incident.
-Further details of specific enforcement policies may be posted separately.
+reported by contacting julianguyen via a private Slack message, or by emailing
```

Comment:
I gave this a second thought, and I'm fine with keeping what we originally had with emailing us. It's easier to keep a record of it. Thanks for thinking to update this!